content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -481,8 +481,8 @@ administrator intervention.
 {{% feature-state for_k8s_version="v1.23" state="alpha" %}}
```

Comment:
You should change this line as well
```suggestion
{{< feature-state feature_gate_name="RecoverVolumeExpansionFailure" >}}
```

Reply:
Yes, I am going to update this and rest of the stuff.